src/js/structs/Item.js

```diff
@@ -14,4 +14,8 @@ module.exports = class Item {
 
     return this._itemData[key];
   }
+
+  toJSON() {
```

Comment:
Yeah, but I'd rather replicate it or introduce a new shared parent class than adding the method to all(!) of our struct.